src/ceph-volume-zfs/ceph_volume_zfs.egg-info/dependency_links.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
egg files/dirs are not supposed to make it into the repo

Reply:
@alfredodeza 
So the whole egg tree should be deleted!? Will do.